### ISSUE TITLE
Don't use bash colors in output if not in interactive session

### ISSUE THREAD

**Issue by gmarek** [kubernetes/test-infra#4628]
I may be mistaken, but I think garbage that's present in our log files are color codes. E.g.
```bash
I0919 05:21:22.074] [90m/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/framework/framework.go:619[0m
```
They make the log file even harder to read. Can we please stop printing them if test is not run from the interactive session?

@kubernetes/sig-testing-bugs @kubernetes/test-infra-maintainers @shyamjvs @wojtek-t

**Comment by shyamjvs** [comment]
+1

**Comment by spxtr** [comment]
cc @rmmh

**Comment by mtaufen** [comment]
According to the [Ginkgo](https://onsi.github.io/ginkgo/) docs, `--noColor` is the appropriate flag.

**Comment by fejta** [comment]
Use gubernator and/or prow to view these logs, which handles them correctly.

**Comment by ixdy** [comment]
I agree with @fejta. We shouldn't disable colors in CI.

**Comment by dims** [comment]
Looks like this is a `wont-fix`hence the revert of #4631

**Comment by gmarek** [comment]
Ekhm - I'm using gubernator to look at those logs, but I always do "Raw build logs", as scalability test logs are, well, large and gubernator historically had problems dealing with them. I'm fine with stripping them from raw build logs view.
